src/assets/styles/_reset.scss

```diff
@@ -0,0 +1,4 @@
+body, h1, h2, h3, h4, h5, h6, p, ul {
+    margin: 0;
+    font-family: 'Mont', sans-serif;
+}
```

Comment:
as mentioned by Anya in my PR: "You don't need to add fonts this way, enough to add it only to body or root"